graph/tags.go

```diff
@@ -127,6 +129,15 @@ func (store *TagStore) reload() error {
 		return err
 	}
 	defer f.Close()
+	finfo, err := f.Stat()
+	if err != nil {
+		return err
+	}
+	// check file is empty and continue with warning
+	if size := finfo.Size(); size == 0 {
+		logrus.Warnf("File empty:%s, restore the original file.", store.path)
```

Comment:
Either that or this should be a real warning like "Tag store is empty, image tags may have been lost"... basically to say that something really went wrong here, there may be data loss.